setup.py

```diff
@@ -579,41 +580,75 @@ def main():
         "build_tests": build_tests,
         "sdist": sdist,
     }
-
-    setup(
-        name="pycairo",
-        version=PYCAIRO_VERSION,
-        url="https://pycairo.readthedocs.io",
-        description="Python interface for cairo",
-        long_description=long_description,
-        maintainer="Christoph Reiter",
-        maintainer_email="reiter.christoph@gmail.com",
-        license="LGPL-2.1-only OR MPL-1.1",
-        ext_modules=[cairo_ext],
-        packages=["cairo"],
-        package_data={
-            "cairo": [
-                "__init__.pyi",
-                "py.typed",
+    if platform.system() == "Windows":
```

Comment:
this will break the normal windows build which doesn't ship a dll, so we'll have to find some other way here

Reply:
maybe an env var that is only used in CI?

Reply:
AFAIK, this doesn't. It just adds `cairo.dll` if it is there inside the specified place or else it just ignores, I tried it ones while building in Travis. I missed copying the `DLL` files inside the directory. It doesn't throw any error. So, there would be no problem.

Reply:
ok. then please don't duplicate the setup call and just special case package_data.